resolwe_bio/processes/alignment/star.yml

```diff
@@ -7,8 +7,10 @@
   name: STAR
   requirements:
     expression-engine: jinja
+    resources:
+      memory: 32768
```

Comment:
@dblenkus At some point we should support defining this with the unit, e.g. 32GB, and update the parser.